src/style.css

```diff
@@ -1 +1,71 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  src: url(fonts/Avenir-Heavy.ttf);
+  src: url(fonts/Avenir-Book.ttf);
```

Comment:
2 times the same URL

Also, you may write @font-face 2 times with different src and font-weight - to explicitly add the font with a higher font-weight (avenir-heavy)

Reply:
change last url to Avenir.ttc;